profiles/sam-configs/terraform/sam/variables.tf.ctmpl

```diff
@@ -0,0 +1,135 @@
+variable "vault_path_prefix" {
+  default = "{{if (env "VAULT_PATH_PREFIX")}}{{ env "VAULT_PATH_PREFIX"}}{{else}}secret/dsde/firecloud/ephemeral/{{env "ENVIRONMENT"}}{{end}}"
```

Comment:
I think this means that vault path prefix has a safe default and doesn't need to be specified in the JSON